contract.md

```diff
@@ -2,39 +2,124 @@
 
 ## Goals
 
-_At a high level, what do you most want to get out of this project, and why?_
+In terms of the project results, I really hope to be able to create at least a
+framework for a language I can use to create more dynamic presentations.
+The main motivation for this project is that I find the current softwares for
+creating presentations limiting and dull, so I would like to be able to make
+something that can be used to create more helpful visuals that flow together
+more seamlessly.
+Additionally, when creating presentations, I can be a bit pedantic about
+whether things are centered or aligned and get frustrated when the software
+requires I drag components with pixel precision to actually make them correctly
+aligned.
+Thus, I hope the language I create can make it easier to place components
+precisely with the desired constraints.
+I hope the language will be similar enough to other web development languages
+(primarily CSS and JavaScript) to make it easy for people familiar with web
+design to understand and use.
+This way, it could be quickly picked up by others with a lower barrier of entry
+than a completely new language, and additionally, it could use syntax that has
+been tested and reviewed by millions over many years which would help ensure
+that it is usable for a wide audience.
+
+In terms of the skills I develop, I primarily want to better hone my ability to
+make a syntax that is intuitive and concise.
+Any language, no matter how powerful, could be effectively useless if the syntax
+isn't something that users can understand and enjoy using.
+Thus, to me this seems like the most important skill to develop in terms of
+making DSLs.
+In addition to developing clean syntax, I hope to improve my ability to convert
+a syntax to my desired output in a straightforward and elegant way.
+I really enjoyed this part of the previous assignment and I hope to explore more
+ways of interpreting structurally complex syntaxes that I may create.
```

Comment:
Is the main point of this language to make it easier for people to program presentations or is it to make presentations more flexible.